modules/lightning_features/lightning_media/modules/lightning_media_image/lightning_media_image.module

```diff
@@ -12,6 +12,24 @@ use Drupal\lightning_core\OverrideHelper as Override;
 use Drupal\lightning_media_image\Plugin\MediaEntity\Type\Image;
 
 /**
+ * Implements hook_modules_installed().
+ */
+function lightning_media_image_modules_installed(array $modules) {
+  if (in_array('lightning_roles', $modules)) {
+    \Drupal::service('lightning.content_roles')
```

Comment:
The grants to media_creator and media_manager should use user_role_grant_permissions(). Only the creator grant should use the lightning.content_roles service.